extension_versions.csv

```diff
@@ -38,7 +38,8 @@ lots,,master,https://raw.githubusercontent.com/open-contracting-extensions/ocds_
 lots,2019-02-25,v1.1.4,https://raw.githubusercontent.com/open-contracting-extensions/ocds_lots_extension/v1.1.4/,https://api.github.com/repos/open-contracting-extensions/ocds_lots_extension/zipball/v1.1.4
 lots,2020-08-20,v1.1.5,https://raw.githubusercontent.com/open-contracting-extensions/ocds_lots_extension/v1.1.5/,https://api.github.com/repos/open-contracting-extensions/ocds_lots_extension/zipball/v1.1.5
 medicine,,master,https://raw.githubusercontent.com/open-contracting-extensions/ocds_medicine_extension/master/,https://github.com/open-contracting-extensions/ocds_medicine_extension/archive/master.zip
-metrics,,master,https://raw.githubusercontent.com/open-contracting-extensions/ocds_metrics_extension/master/,https://github.com/open-contracting-extensions/ocds_metrics_extension/archive/master.zip
+metrics,,1.1,https://raw.githubusercontent.com/open-contracting-extensions/ocds_metrics_extension/1.1/,https://github.com/open-contracting-extensions/ocds_metrics_extension/archive/1.1.zip
+metrics,,1.2,https://raw.githubusercontent.com/open-contracting-extensions/ocds_metrics_extension/1.2/,https://github.com/open-contracting-extensions/ocds_metrics_extension/archive/1.2.zip
```

Comment:
```suggestion
```

OCDS 1.2 is not released. Similarly, 1.2 versions of extensions are not released.